.sphinx/_static/custom.css

```diff
@@ -193,3 +193,26 @@ details summary {
 .highlight button.copybtn {
    opacity: 0;
 }
+
+/* Mimicking the 'Give feedback' button for UX consistency */
```

Comment:
This button has black text now, while the other one has white. We should be consistent.
I think the white text is easier to read, and it fits better with the light theme (and both work for the dark theme).

Reply:
This is what I see on mine:
![image](https://github.com/canonical/sphinx-docs-starter-pack/assets/116022293/9978646f-977a-4132-81c5-437f3030f04e)

The contrast of black on orange is VERY low, I would strongly suggest making it white for better visibility.

Reply:
Thanks for the update, looks nice now!

![image](https://github.com/canonical/sphinx-docs-starter-pack/assets/116022293/1504c62b-6ae5-426a-b1e8-cae913689ef2)